src/mono/msbuild/common/LibraryBuilder.targets

```diff
@@ -35,4 +41,35 @@
     </LibraryBuilderTask>
   </Target>
 
+  <Target Name="_GenerateBundle"
+          Condition="'$(BundlesAssemblies)' == 'true'">
```

Comment:
Since the bundle contains various resources, do we want also rename the `BundlesAssemblies` and the rest of the "assemblies" here?

Reply:
yep! Thanks for the catch